Composer/packages/server/src/router/api.ts

```diff
@@ -68,7 +68,7 @@ router.post('/runtime/eject/:projectId/:template', EjectController.eject);
 //assets
 router.get('/assets/projectTemplates', AssetController.getProjTemplates);
 
-router.use('/assets/locales/', express.static(path.join(__dirname, '..', '/locales')));
+router.use('/assets/locales/', express.static(path.join(__dirname, '../../src/locales')));
```

Comment:
I think it would help to break this out as `'..','..','src','locales'`, just so we're not assuming the OS's path separators.